product-updates/knowledge-center/view-submission-history.html

```diff
@@ -308,30 +308,157 @@ <h3 class="usa-process-list__heading">Search for the reporting period to view</h
+
+   <h2>Download Error Reports and Previously Submitted Data Files</h2>
+  <div style="min-height: 22px;"></div>
+    
+
+      <p>To access error reports for each file you submit. Click on the link in the ‘Error Report’ column to view errors and data feedback associated with that version of the file. Depending on the completeness of your data and how soon you access the Submission History tab after submitting a file you may see other values in this column including "Pending" and "No Errors".</p>
+  <div style="min-height: 22px;"></div>
+       <p>To access a file previously uploaded to TDP from 'Submission History', you can click the link in the ‘File Name’ column. Note: TDP will auto-rename files to specify the fiscal period and section (e.g. 2021.Q1.Active Case Data), so depending on your original file-naming conventions, the files you download may have a different name.</p>
+         <div style="min-height: 22px;"></div>
+      <a href="../img/getting-started/click-file-name-download.png" data-lity data-lity-desc="Submission history table with the file name column emphasized">
+      <img src="../img/getting-started/click-file-name-download.png" alt="Submission history table with the file name column emphasized">
+      </a>
+
+<hr style="border: none;
+    height: 1px;
+    background: #000;
+    margin-top: 32px;
+    background: repeating-linear-gradient(90deg,#000,#000 6px,transparent 6px,transparent 12px);">
+    <div style="min-height: 22px;"></div> <div style="min-height: 22px;"></div> 
+
+
+
+
+
+         <h2>Understanding File Statuses and Aggregate Data</h2>
+               <div style="min-height: 22px;"></div> 
+                     <p>The Submission History table includes a status which communicates the level of completeness of each file and related aggregate data about its contents including counts of cases with and without errors, counts of records unable to process within a file, and (for section 3 and 4 data) a count of total errors in the file.</p>
+                      <div style="min-height: 22px;"></div> 
+      <a href="../img/getting-started/submission-history-tables.png" data-lity data-lity-desc="Submission history table with the error report column emphasized">
+      <img src="../img/getting-started/submission-history-tables.png" alt="Submission history table with aggregate values and the status emphasized">
+      </a>
+  <div style="min-height: 22px;"></div>
+
+
+
+       <div style="min-height: 22px;"></div>
+
+  <table class="usa-table">
+  <caption>
+   File Statuses Key
+  </caption>
+  <thead>
+    <tr>
+      <th scope="col">Status</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row"><img src="../img/alerts/clock.svg" alt="" aria-hidden="true" style="width: 20px; height: 20px; position: relative; top: 4px; margin-right: 5px;">Pending</th>
+      <td>
+        Your file is being processed by TDP.
```

Comment:
also, in some circumstances:

- if pending status persist more than 4-5minutes, there may be an issue processing the file
- especially for files submitted prior to February 2024, pending status means that the file was submitted before data parsing logic was added to the system. In this case, no action is needed from the user at this time.